### PR TITLE
add spanish translations to some sentences

### PR DESCRIPTION
**Pull request by forerosantiago**
Hey, I translated a lot of sentences to Spanish (:

### CODE REVIEW

**Review by flashcode — CHANGES_REQUESTED** [review]
Hi,

Thanks for the update on translations.
There are some errors (see my comments), and please ensure the file is OK using https://github.com/flashcode/msgcheck.

**Comment by forerosantiago** [comment]
All right @flashcode, I corrected the mistakes you pointed and checked the po file with msgcheck, the file is OK.

**Comment by flashcode** [comment]
Hi,

Thanks for changes.

Many "fuzzy" markers are still there on translations you updated, you must remove them if the translation is done, otherwise the translation is not used at all (considered wrong or approximative, so English is used instead).

Also could you please rebase your branch on master, by merging your changes with the latest changes in WeeChat? (maybe using command `msgmerge`), because now there are conflicts.

**Comment by forerosantiago** [comment]
All right, Iremoved several fuzzy markers and tried to fix the conflicts, can you please confirm if I fixed the conflicts correctly?

**Comment by forerosantiago** [comment]
Oops, accidentally closed.

File: po/es.po [hunk 1]
```diff
@@ -630,48 +655,48 @@ msgstr[1] "%d nuevos atajos agregados (contexto: \"%s\")"
 
 #, fuzzy
```
Comment (flashcode) [inline po/es.po:656]: Hi,
Some translations are still marked as "fuzzy" but translated, you must remove this fuzzy marker (multiple places).
Thanks.

**Comment by forerosantiago** [comment]
Hello @flashcode, I fixed the errors and made a commit a few days ago. Just reminding you about it in case you missed it.

Cheers!

**Review by flashcode — APPROVED** [review]

**Comment by flashcode** [comment]
Thanks, I upported your changes on the master branch and committed your changes.

**Pull request closed without merging**